lib/KeyGenUtils.py

```diff
@@ -1,91 +1,158 @@
-import uuid
+import sys
+import os
+sys.path.append(os.path.abspath(os.path.dirname(__file__) + '/' + './'))
+
 import getpass
-import Checksum
+import uuid
+from Crypto.Cipher import AES
+import json
+import hashlib
+import random
+import string
+import base64
+from constants import KeyGenConst
+
 
-# Every Time Key will be different but gets verify only from the source it generated.
 # Key have the length of 108 characters ends with '=' symbol.
 # Please install pycryptodome before run this file. Use cmd 'pip install pycryptodome'.
 # Change the folder to Crypto from crypto.
+# Generated Key = (plain text + SALT) -> sha256_encryption -> AES encryption
+# Verify Key = (provided key -> AES decryption) campare with ((plain text + SALT) -> sha256_encryption)
 
 
+# return MAC address @return type String
 def __get_mac__():
     mac = hex(uuid.getnode())
     return mac
 
+
+# return system user login id @return type String
 def __get_current_user__():
     username = getpass.getuser()
     return username
 
 
-#One System -> One Key
+# One System -> One Key @return type String
 def get_key_by_system():
     params = {
         'MAC': str(__get_mac__())
     }
-    checksum = Checksum.generate_checksum(params, b'Iy9UErx4chg4fdA6')  #Key used for encryption
-    return checksum
+    json_string = json.dumps(params)
+    key = __generate_key__(json_string)
+    return key
 
 
-#One User -> One Key
+# One User -> One Key @return type String
 def get_key_by_username():
     params = {
         'USERNAME': str(__get_current_user__())
     }
-    checksum = Checksum.generate_checksum(params, b'Iy9UErx4chg4fdA6', '45')  #Key used for encryption
-    return checksum
+    json_string = json.dumps(params)
+    key = __generate_key__(json_string)
+    return key
 
 
-#One User and One System -> One Key
+# One User and One System -> One Key @return type String
 def get_key_by_username_and_system():
     params = {
-        'USERNAME'  : str(__get_current_user__()),
-        'MAC'       : str(__get_mac__())
+        'USERNAME': str(__get_current_user__()),
+        'MAC': str(__get_mac__())
     }
-    checksum = Checksum.generate_checksum(params, b'Iy9UErx4chg4fdA6')  #Key used for encryption
-    return checksum
+    json_string = json.dumps(params)
+    key = __generate_key__(json_string)
+    return key
 
 
+# @return type boolean
 def verify_key_by_system(key):
     params = {
         'MAC': str(__get_mac__())
     }
-    isVerified = Checksum.verify_checksum(params, b'Iy9UErx4chg4fdA6', key)
+    json_string = json.dumps(params)
+    isVerified = __verify_key__(json_string, key)
     return isVerified
 
 
+# @return type boolean
 def verify_key_by_username(key):
     params = {
         'USERNAME': str(__get_current_user__())
     }
-    isVerified = Checksum.verify_checksum(params, b'Iy9UErx4chg4fdA6', key, '45')
+    json_string = json.dumps(params)
```

Comment:
camel casing, means remove underscores, and write the name as jsonString instead of json_string. Change this for all.